Description: Fix compile time warning about too many arguments for format
Author: James Troup <james.troup@canonical.com>
--- a/nicstat.c
+++ b/nicstat.c
@@ -1544,7 +1544,7 @@ load_snmp(FILE *snmp)
 			n = fscanf(snmp, "Udp: %lld %lld %lld %lld "
 			    "%lld %lld\n",
 			    &ll[0], &ll[1], &ll[2], &ll[3],
-			    &ll[4], &ll[5], &ll[6]);
+			    &ll[4], &ll[5]);
 			if (n == 6) {
 				g_udp_new->inDatagrams = ll[0];
 				g_udp_new->outDatagrams = ll[3];
